lib/ddtrace/contrib/http/instrumentation.rb

```diff
@@ -37,22 +36,17 @@ module InstanceMethods
           def request(req, body = nil, &block)
             host, = host_and_port(req)
             request_options = datadog_configuration(host)
-            pin = datadog_pin(request_options)
-            return super(req, body, &block) unless pin
+            client_config = Datadog::Tracing.configuration_for(self)
```

Comment:
I've seen a similar change for elasticsearch and other integrations -- that we assume that there's always going to be a `pin` for the integration, rather than checking. Is this deliberate / OK?

Reply:
Presence of a `pin` isn't required; it's an optional override, which is preferred if present.